### ISSUE TITLE
OCSInventory-Server don't write log

### ISSUE THREAD

**Issue by Mant1kor** [OCSInventory-NG/OCSInventory-Server#100]
### General informations
Operating system :  CentOS Linux release 7.4.1708 (Core)

### Server informations
Perl version :  Perl/v5.16.3
Mysql / Mariadb / Percona version :  MySQL Community Server (GPL) version 5.7.21

### Server installation method ( Only one choice )
- [ ] RPM Package
- [x] Install SH
- [ ] Manual installation

### OCS Inventory informations
Ocs server version :  2.4

### Problem's description
I've installed OCSInventory 2 weeks ago. It's have 5 test clients which send info every 24h, but logs folder is empty.

z-ocsinventory-server.conf

```
  # Path to log directory (must be writeable)
  PerlSetEnv OCS_OPT_LOGPATH "/var/log/ocsinventory-server"
```

```
$ ls -laZ /var/log/ | grep ocsinventory
drwxrwxr-x. apache apache unconfined_u:object_r:var_log_t:s0 ocsinventory-server
```

**Comment by Mant1kor** [comment]
For some reason it began to write a log from 17.04.18 ¯\_(ツ)_/¯

**Comment by TiVince** [comment]
I see a dot after the permission that means you are in SELINUX context. You have to configure Apache to be able to write in the log file

**Comment by gillesdubois** [comment]
This problem has been addressed in the 2.5 / 2.6RC. The log are written to the directory without any problem. 

Thx for your contribution.

Regards,
Gilles Dubois.
